### DISCUSSION TITLE
Нормальные штрафы.

### DISCUSSION THREAD

**Discussion by An1reyka** [octothorp-team/public-dobrograd#686] [Предложения]
Возможно эта тема уже когда-то давно предлагалась, но все же. Думаю всех бесит что нет нормальной системы штрафов и когда на тебе тикет ты не можешь поехать на остоновке и на тебя срабатывают все камеры. Предлагаю сделать нормальную систему выдачи штрафов, примерно так же как и розыск только чтобы на него камеры не реагировали. Также если это можно реализовать, можно добавить если по истечению срока который ведет офицер человек не оплатит тикет, на него вешался розыск по причине неуплаты, а там уже и легкий срок.

**Comment by Darling-in-Balbes** [comment]
Уже в планах
